src/test/kotlin/io/github/mojira/arisa/modules/ReplaceTextModuleTest.kt

```diff
@@ -0,0 +1,283 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.right
+import io.kotest.assertions.arrow.either.shouldBeLeft
+import io.kotest.assertions.arrow.either.shouldBeRight
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class ReplaceTextModuleTest : StringSpec({
+    val module = ReplaceTextModule()
+    "should return OperationNotNeededModuleResponse when there is no description nor comment" {
+        val request = ReplaceTextModule.Request(
```

Comment:
[Minor] I generally like to add imports for these so you can simply use `Request`, or `Comment` (Alt + Enter, add Import)